gobblin-core/src/main/java/gobblin/source/extractor/filebased/FileBasedExtractor.java

```diff
@@ -192,4 +209,14 @@ public void close() {
       LOGGER.error("Could not successfully close file system helper due to error: " + e.getMessage(), e);
     }
   }
+
+  private void incrementBytesReadCounter() {
+    try {
+      counters.inc(CounterNames.FileBytesRead, fsHelper.getFileSize(currentFile));
+    } catch (FileBasedHelperException e) {
+      LOGGER.info("Unable to get file size. Will skip increment to bytes counter " + e.getMessage());
```

Comment:
You can do `LOGGER.info("Unable to get file size. Will skip increment to bytes counter ", e);`. It will print the entire stacktrace, but it is more informative.